src/test/java/org/broadinstitute/consent/http/resources/UserResourceTest.java

```diff
@@ -85,54 +81,37 @@ public class UserResourceTest {
 
   private final String TEST_EMAIL = "test@gmail.com";
 
-  private AuthUser authUser;
-
-  @BeforeEach
-  public void setUp() throws URISyntaxException {
-    authUser = new AuthUser()
-        .setAuthToken("auth-token")
-        .setName("Test User")
-        .setEmail(TEST_EMAIL)
-        .setUserStatusInfo(userStatusInfo);
-    openMocks(this);
-    when(uriInfo.getRequestUriBuilder()).thenReturn(uriBuilder);
-    when(uriBuilder.path(anyString())).thenReturn(uriBuilder);
-    when(uriBuilder.build(anyString())).thenReturn(new URI("http://localhost:8180/dacuser/api"));
-  }
+  private final AuthUser authUser = new AuthUser()
+      .setAuthToken("auth-token")
+      .setName("Test User")
+      .setEmail(TEST_EMAIL)
+      .setUserStatusInfo(userStatusInfo);
 
   private void initResource() {
     userResource = new UserResource(samService, userService, datasetService,
         acknowledgementService);
   }
 
   @Test
-  public void testGetMe() {
+  void testGetMe() {
     User user = createUserWithRole();
     when(userService.findUserByEmail(any())).thenReturn(user);
-    when(userService.findAllUserProperties(any())).thenReturn(createResearcherProperties());
-    when(libraryCardService.findLibraryCardsByUserId(any()))
-        .thenReturn(createLibraryCards());
```

Comment:
Can you briefly describe why these conditions (in this method and others) are no longer necessary to test?

Reply:
Including them causes Mockito to complain that they are unnecessary stubs. It's not that they aren't under test, it's that their return values are irrelevant to the test method.